oauth2client/crypt.py

```diff
@@ -248,3 +237,25 @@ def verify_signed_jwt_with_certs(jwt, certs, audience=None):
     _check_audience(payload_dict, audience)
 
     return payload_dict
+
+
+def configure_module():
+    """Configure module-wide variables.
+
+    Not all testing frameworks load modules directly, so we need a way to
+    define these on set up.
+    """
+    global Signer
+    global Verifier
```

Comment:
The issue isn't the use of globals / module level constants, just the use of the `global` keyword. It makes the module harder to understand.